src/main/java/com/microsoft/azure/spring/data/cosmosdb/core/DocumentDbOperations.java

```diff
@@ -69,4 +67,6 @@
     <T> Observable<T> deleteAllAsync(String collectionName, List<String> partitionKeyNames);
 
     <T> Observable<Page<T>> findAllAsync(Pageable pageable, Class<T> domainClass, String collectionName);
+
+    <T> Observable<T> findAllAsync(String collectionName, Class<T> entityClass, String partitionKeyName);
```

Comment:
I see both `entityClass` and `domainClass` are used in this interface. What's story behind it?

Reply:
legacy naming, will renaming all to entityClass in another cleanup PR.